src/common/Version.cpp

```diff
@@ -10,8 +10,31 @@ namespace chatterino {
 
 using namespace literals;
 
+/**
+ * Valid version formats, in order of latest to oldest
+ *
+ * Stable:
+ *  - 2.4.0
+ *
+ * Release candidate:
+ *  - 2.4.0-rc.3
+ *  - 2.4.0-rc.2
+ *  - 2.4.0-rc
+ *
+ * Beta:
+ *  - 2.4.0-beta.3
+ *  - 2.4.0-beta.2
+ *  - 2.4.0-beta
+ *
+ * Alpha:
+ *  - 2.4.0-alpha.3
+ *  - 2.4.0-alpha.2
+ *  - 2.4.0-alpha
+ **/
+constexpr QStringView CHATTERINO_VERSION = u"2.5.1";
```

Comment:
Why not keep this defined in the header and continue to use `CHATTERINO_VERSION` everywhere?

Reply:
Done.